### PR TITLE
[#165008942] service detail screen markdown loading

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to fix the case when markdown content is rendered during the ease-out animation of the Activity indicator as shown here

screen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#165008942](https://www.pivotaltracker.com/story/show/165008942): Layout della scheda servizio del progetto IO corrotto

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/components/ui/Markdown/index.tsx [hunk 1]
```diff
@@ -206,40 +206,41 @@ class Markdown extends React.PureComponent<Props, State> {
...
-            />
-          </View>
-        </ScrollView>
-      );
-    }
-
-    return <ActivityIndicator />;
+    return (
+      <React.Fragment>
+        <ActivityIndicator
```
Comment (cloudify) [inline ts/components/ui/Markdown/index.tsx]: `ActivityIndicator` is always rendered?

Comment (fpersico) [reply ts/components/ui/Markdown/index.tsx]: animating=false hide it.

Comment (lmorelli986) [reply ts/components/ui/Markdown/index.tsx]: Yes, but the `animating` property is responsible of hiding it when unnecessary.

**Pull request merged**